uiowa/uiowa
# Recreate existing view page as a view block that can be placed on any layout builder page. #4069

Closes #4069 


# How to test
1. Checkout branch
2. `blt ds --site=default`
3. `drush @default.local uli`
4. activate areas_of_study split 
5. `drush @default.local cim`
6. `blt frontend`
6. `drush @default.local cr`
7. Go to a page -> layout builder -> add 'Areas of Study' block to page
8. Make sure displays purrrrtttyyy and functions properly
